vault/resource_password_policy_test.go

```diff
@@ -0,0 +1,58 @@
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/hashicorp/vault/api"
+)
+
+func TestAccPasswordPolicy(t *testing.T) {
```

Comment:
Can you add a test to ensure multiple charsets work?

Reply:
Done! 👍